test/lit.cfg

```diff
@@ -254,6 +254,16 @@ native_swift_tools_path = lit_config.params.get('native_swift_tools_path')
 if native_swift_tools_path is not None:
     append_to_env_path(native_swift_tools_path)
 
+libswift_build_mode = lit_config.params.get('libswift_build_mode')
+if libswift_build_mode != 'DISABLE':
+  config.available_features.add('libswift')
+  if libswift_build_mode == 'HOSTTOOLS':
+    config.available_features.add('libswift_hosttools')
```

Comment:
I think I cherry-picked too hard right here. I am going to fix this part in a subsequent commit.